CMakeLists.txt

```diff
@@ -34,7 +34,24 @@ add_definitions(-DNOMINATIM_VERSION="${NOMINATIM_VERSION}")
 
 set(BUILD_TESTS off CACHE BOOL "Build test suite" FORCE)
 set(WITH_LUA off CACHE BOOL "Build with lua support" FORCE)
-add_subdirectory(osm2pgsql)
+set(WWW_PUBLIC /var/www/nominatim CACHE PATH "Public www root")
+set(WWW_APP ${CMAKE_INSTALL_PREFIX}/share/nominatim CACHE PATH "Web application root")
+find_program(OSM2PGSQL osm2pgsql DOC "System installed osm2pgsql")
```

Comment:
This is not a good idea. Nominatim is bound to a very specific version of osm2pgsql. Running it with any other than the one that comes with the package is not supported. I'm aware that this is a problem for packagers. How about renaming the osm2pgsql built by Nominatim into nominatim-osm2pgsql, so that it can exist in parallel with another installed version.